.github/workflows/super-linter.report

```diff
@@ -0,0 +1 @@
+
```

Comment:
Is this file necessary for some reason?

Reply:
It reports an error about the file missing. You can see it in the build steps. So I created it to get written to the next time it runs.

Reply:
Wait does it though? Or am I looking at the wrong thing here?

```
2020-09-15 15:47:52 [INFO  ]   Generated reports:
2020-09-15 15:47:52 [INFO  ]   ----------------------------------------------
2020-09-15 15:47:52 [INFO  ]   ----------------------------------------------
2020-09-15 15:47:52 [INFO  ]   Reports generated in folder /github/workspace/super-linter.report
2020-09-15 15:47:52 [WARN  ]   Report output folder (/github/workspace/super-linter.report) does NOT exist.
```

That sounds like it wants a directory for that path, not a file. Also, it looks like the path should be `github/workspace/super-linter.report`, not `github/workflows/super-linter.report`.


https://github.com/github/super-linter seems to cocnur that it's supposed to be a path to a folder:

> ENV VAR | Default Value | Notes
> -- | -- | --
> OUTPUT_FOLDER | super-linter.report | The location where the output reporting will be generated to. Output folder must not previously exist.


But maybe I'm not understanding how that part works

Reply:
Ah ok. I'll remove the file. I guess I didn't understand that part correctly.